File: pertemuan 5/tugas5.java
import java.util.Scanner;

public class tugas5 {
    public static void main(String[] args){
        Scanner IPK = new Scanner(System.in);        
            System.out.print("berapakah IPK anda : " );
            double ipk = IPK.nextDouble();
            System.out.print("berapa tahun anda menempuh masa kuliah anda sampai lulus : " );
            int masa = IPK.nextInt();

            if (ipk >= 3.51 && ipk <= 4.00 && masa <= 4 && masa > 2) {
            System.out.println("anda mendapatkan predikat summa cumlaude");
            } else if (ipk >= 3.51 && ipk <= 4.00 && masa > 4){
                System.out.println("anda mendapatkan predikat cumlaude");
            } else if (ipk >= 3.01 && masa >=3 && ipk <= 4.00){
                System.out.println("anda mendapatkan predikat sangat memuaskan");
            } else if (ipk >= 2.76 && masa >= 3 && ipk <= 4.00){
                System.out.println("anda mendapatkan predikat memuaskan");
            } else if (ipk >= 2.00 && masa >=3 && ipk <= 4.00){
                System.out.println("anda mendapatkan predikat cukup");
            } else if (ipk < 2.00 && masa >=3 && ipk <= 4.00){
                System.out.println("mohon maaf, anda tidak lulus");
            } else {
                System.out.println("mohon maaf, data yang anda masukkan salah");
            }
        }
    }

File: pertemuan 5/tugas5.py
ipk = input("berapakah IPK anda : ")
ipk = float(ipk)
masa = input("berapa tahun anda menempuh masa kuliah anda : ")
masa = int(masa)

if (ipk >= 3.51 and ipk <= 4.00 and masa >= 2 and masa <= 4):
    print("anda mendapatkan predikat summa cumlaude")
elif (ipk >= 3.51 and ipk <= 4.00 and masa >= 4 and masa <= 10):
    print("anda mendapatkan predikat cumlaude") 
elif (ipk >= 3.01 and masa >=3 and ipk <= 4.00):
    print("anda mendapatkan predikat sangat memuaskan")
elif (ipk >= 2.76 and masa >= 3 and ipk <= 4.00):
    print("anda mendapatkan predikat memuaskan")
elif (ipk >= 2.00 and masa >=3 and ipk <= 4.00):
    print("anda mendapatkan predikat cukup")
elif (ipk < 2.00 and masa >=3 and ipk <= 4.00):
    print("mohon maaf, anda tidak lulus")
else:
    print("maaf, data yang anda masukkan salah")